src/commands/payid-verify.ts

```diff
@@ -0,0 +1,42 @@
+import { PaymentInformation, verifyPayId } from '../verifiable'
+import { convertJsonToVerifiedAddress } from '../verifiable/converters'
+
+import Command from './Command'
+
+/**
+ * Verifies the signatures and certs for verified addresses of the currently loaded PayID.
+ */
+export default class VerifyPayIdCommand extends Command {
+  protected async action(): Promise<void> {
+    const info = this.getPaymentInfo()
+    if (verifyPayId(info)) {
+      const addresses = info.verifiedAddresses.map((address) => {
+        return convertJsonToVerifiedAddress(address.payload)
```

Comment:
So this function does the verification and throws an error if it fails?

I'm not sure from glancing over this file how we actually verify the PayID.

Reply:
verifyPayId is a check that all verified address check out. You'll see in the next PR that I added an "inspect" command because you probably want to see more detail into all verified addresses and signatures.